cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/BasicPushStrategy.java

```diff
@@ -74,25 +75,22 @@
 
     private static JSONHelper sJsonHelper = new JSONHelper();
 
-    public BasicPushStrategy(PushReplication pushReplication) {
-        this(pushReplication, null);
-    }
+    public int changeLimitPerBatch = 500;
```

Comment:
See https://github.com/cloudant/sync-android/blob/master/cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/PullConfiguration.java#L23 and https://github.com/cloudant/sync-android/blob/master/cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/PushConfiguration.java#L23

Maybe somebody thought 500 was better suited to the internal SQLite database and 1000 to a remote CouchDB? I think it's fairly arbitrary and I just brought those values over rather than changing them.